### PR TITLE
Adds runner file and StatTracker class

### PR DESCRIPTION
**Pull request by drewwilliams5280**
Setup and I/O complete.

### CODE REVIEW

**Review by coreycartercodes — COMMENTED** [review]
Comment rating 3/5 stars

**Comment by abcdefghijohn** [comment]
Good work but I feel you're not utilizing your full potential in comment writing. I know you've got way more in you Drew, I see it in your eyes.

**Review by abcdefghijohn — APPROVED** [review]
Looks good

**Comment by TravisGM92** [comment]
I'll say; I like your comment Drew. You're killing it.

File: lib/runner.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+game_path = './data/games.csv'
+team_path = './data/teams.csv'
+game_teams_path = './data/game_teams.csv'
+
+locations = {
+  games: game_path,
+  teams: team_path,
+  game_teams: game_teams_path
+}
```
Comment (TravisGM92) [inline lib/runner.rb:11]: This'll be interesting... I'm looking forward to seeing all of our ideas on how to access the 'locations' hash

**Review by coreycartercodes — APPROVED** [review]
For initial setup

**Pull request merged**